packages/@aws-cdk/aws-redshift/lib/table.ts

```diff
@@ -297,6 +312,18 @@ export class Table extends TableBase {
     const sortKeyColumns = getSortKeyColumns(columns);
     return (sortKeyColumns.length === 0) ? TableSortStyle.AUTO : TableSortStyle.COMPOUND;
   }
+
+  private validateColumnIds(columns: Column[]): void {
+    const columnIds = new Set<string>();
+    for (const column of columns) {
+      if (column.id) {
+        if (columnIds.has(column.id)) {
+          throw new Error(`Column id '${column.id}' is not unique.`);
+        }
+        columnIds.add(column.id);
```

Comment:
`validate` sounds like an operation with no side effects, but this line performs a side effect. Can we change the method name, something like `addColumns()` or `addColumnIds()`?

Reply:
This has been done